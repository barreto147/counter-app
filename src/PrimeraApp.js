const PrimeraApp = ({ saludo = "Hola mundo" }) => {
  return (
    <>
      <h1>{saludo}</h1>
      <p>Este es un componente</p>
    </>
  );
};

export default PrimeraApp;
