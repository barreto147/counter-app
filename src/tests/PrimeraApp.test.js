import { render } from "@testing-library/react";
import PrimeraApp from "../PrimeraApp";
import { shallow } from "enzyme";

describe("Pruebas en <PrimeraApp/>", () => {
  //   test('Debe mostrar el mensaje "Hola, Soy Coku"', () => {
  //     const saludo = "Hola, Soy Coku";
  //     const { getByText } = render(<PrimeraApp saludo={saludo} />);
  //     expect(getByText(saludo)).toBeInTheDocument();
  //   });

  test("Debe mostrar <PrimeraApp/> correctamente", () => {
    const saludo = "Hola, Soy Coku";

    const wraper = shallow(<PrimeraApp saludo={saludo} />);

    expect(wraper).toMatchSnapshot();
  });
});
