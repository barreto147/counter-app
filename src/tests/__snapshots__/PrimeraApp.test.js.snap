// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp/> Debe mostrar <PrimeraApp/> correctamente 1`] = `
<Fragment>
  <h1>
    Hola, Soy Coku
  </h1>
  <p>
    Este es un subtitulo
  </p>
</Fragment>
`;
